diff -urN aa/streamtuner-0.12.4/help/C/Makefile.am streamtuner-0.12.4/help/C/Makefile.am
--- aa/streamtuner-0.12.4/help/C/Makefile.am	2004-07-26 17:24:31.000000000 +0200
+++ streamtuner-0.12.4/help/C/Makefile.am	2004-07-29 00:25:37.691504728 +0200
@@ -4,7 +4,7 @@
 omffile = streamtuner-C.omf
 entities = documentation-license.xml software-license.xml
 
-include $(top_srcdir)/help/xmldocs.make
+include $(top_srcdir)/xmldocs.make
 
 dist-hook: app-dist-hook
 	SGML_CATALOG_FILES=/usr/local/share/sgml/catalog xmllint --noout --valid --catalogs $(docname).xml
