diff -urN aa/streamtuner-0.12.4/data/streamtuner.desktop.in streamtuner-0.12.4/data/streamtuner.desktop.in
--- aa/streamtuner-0.12.4/data/streamtuner.desktop.in	2004-07-23 20:04:08.000000000 +0200
+++ streamtuner-0.12.4/data/streamtuner.desktop.in	2004-07-29 14:00:09.750676376 +0200
@@ -8,5 +8,5 @@
 TryExec=streamtuner
 Exec=streamtuner
 Terminal=false
-Categories=Application;AudioVideo;GTK;Tuner;
+Categories=AudioVideo;GTK;Tuner;
 StartupNotify=true
